Transfer/Sources/CreateTransferController.swift

```diff
@@ -437,6 +438,10 @@ extension CreateTransferController: CreateTransferView {
             )
         }
     }
+
+    func dismissKeyboard() {
+        view.endEditing(true)
+    }
```

Comment:
Why need this? `hideKeyboardWhenTappedAround` didn't work?

Reply:
I did not try that, let me check that too.